src/main/config/messages_template.xml

```diff
@@ -423,6 +423,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ085E" type="ERROR">
+    <reason>Parameter '%1' cannot be set with param in project file. The param '%1' will be ignored.</reason>
+    <response>Use '%2' key instead.</response>
```

Comment:
@infotexture Could you check the copy on this error message. Because we have multiple project formats and some parameters in XML are defined with elements and some are defined with attributes, the response can't be e.g. "Use <%2> element instead.". But _key_ is not a particularly good term for this either.

Reply:
The error messages will not be

```
Error: [DOTJ085E][ERROR] Parameter 'output.dir' cannot be set with param in project file. The param 'output.dir' will be ignored. Use 'output' key instead.
Error: [DOTJ085E][ERROR] Parameter 'args.filter' cannot be set with param in project file. The param 'args.filter' will be ignored. Use 'profiles' key instead.
Error: [DOTJ085E][ERROR] Parameter 'args.input' cannot be set with param in project file. The param 'args.input' will be ignored. Use 'input' key instead.
Error: [DOTJ085E][ERROR] Parameter 'transtype' cannot be set with param in project file. The param 'transtype' will be ignored. Use 'transtype' key instead.
```

Reply:
Could this be parameterized as

```
Use the %s instead.
```

where `%s` could be `the <output> element` or `the @transtype attribute`?

Reply:
Currently we don't have support for something like that because the whole idea of messages is that the code doesn't contain any of the copy.